src/main/java/util/Ui.java

```diff
@@ -0,0 +1,118 @@
+package util;
+
+import task.Task;
+
+/**
+ * The UI class deals with interactions with the user and prints responses accordingly.
+ */
+public class Ui {
+    /**
+     * Returns the welcome message.
+     *
+     * @return returns the welcome message.
+     */
+    public String showWelcome() {
+        return "Serina here, how can I assist you? Enter 'help' to learn what I can do for you.";
+    }
+
+    /**
+     * Prints the help message with instructions on how to use Serina.
+     */
+    public String showHelp() {
+        return "I am Serina, your personal toDo Manager. These are the commands you can issue me: " + "\n"
+                + "1. list" + " => lists all your current tasks " + "\n"
+                + "3. todo {task description} => creates a new to do task with given description " + "\n"
+                + "4. deadline {task description} /by {YYYY-MM-DD HHMM} => creates a new task with given "
+                + "description and deadline" + "\n"
+                + "5. event {task description} /at {YYYY-MM-DD HHMM} => creates a new task with given "
+                + "description and event time" + "\n"
+                + "6. done {task number} => marks the given task as done" + "\n"
+                + "7. delete {task number} => deletes the given task" + "\n"
+                + "8. find {query} => lists out tasks based on your query" + "\n"
+                + "9. bye => makes me go to sleep";
+    }
+
+    /**
+     * Returns the string representation of a task.
+     *
+     * @param task Task to print
+     * @param taskNum Task number of task
+     * @return string representation of a task.
+     */
+    public String showTask(Task task, int taskNum) {
+        return taskNum + ". " + task;
+    }
+
+    /**
+     * Returns the add task acknowledgement string.
+     *
+     * @param task    Task to print
+     * @param taskNum Task number of task
+     * @return  Add task acknowledgement string.
+     */
+    public String showAddTask(Task task, int taskNum) {
+        return "The following task has been added: \n" + this.showTask(task, taskNum);
+    }
+
+    /**
+     * Returns the delete task acknowledgement string.
+     *
+     * @param task    Task to print
+     * @param taskNum Task number of task
+     * @return The delete task acknowledgement.
+     */
+    public String showDeleteTask(Task task, int taskNum) {
+        return "The following task has been deleted: \n" + this.showTask(task, taskNum);
+    }
+
+    /**
+     * Returns the task mark as done acknowledgement string.
+     *
+     * @param task    Task to print
+     * @param taskNum Task number of task
+     * @return The task mark as done acknowledgement string.
+     */
+    public String showDoneTask(Task task, int taskNum) {
+        return "The following task has been marked as done: \n" + this.showTask(task, taskNum);
+    }
```

Comment:
Perhaps showAddTaskMessage, showDeleteTaskMessage and showDoneTaskMessage would be more descriptive and less ambiguous

Reply:
Hmm I get what you mean but I think i prefer to keep the method names shorter!